agent/README.md

```diff
@@ -54,3 +54,15 @@ the recommended configurations through some concrete examples.
 | dev/qe |  | In this case the latest _nightly_ release is automatically downloaded, and the installer is extracted<br>from that payload. `OPENSHIFT_RELEASE_STREAM` and `OPENSHIFT_RELEASE_TYPE` respectively<br>are used to determine the version and stream to be gathered |
 | dev/qe | `OPENSHIFT_RELEASE_IMAGE=`<br>`registry.ci.openshift.org/ocp/release:4.11.0-0.nightly-2022-05-11-054135` | As before, but pinning to a specific release version |
 | CI | `OPENSHIFT_RELEASE_IMAGE=<ephemeral payload pullspec>`<br>`OPENSHIFT_CI=true` | This is the configuration used in the CI to test an ephemeral payload |
+
+# Troubleshooting
+
+If `make agent` target fails booting the ISO due to permission issue, for example,
```

Comment:
It's probably worth mentioning that this could happen on those systems where libvirtd was just installed, and it's required only once